inc/deployfile.class.php

```diff
@@ -1056,36 +1066,53 @@ static function addFileInRepo($params) {
     * Remove file from the repository
     *
     * @param string $sha512 sha512 of the file
-    * @param integer $packages_id id of the package
     * @return boolean
     */
-   static function removeFileInRepo($sha512, $packages_id) {
+   static function removeFileInRepo($sha512) {
+      $repoPath        = GLPI_PLUGIN_DOC_DIR."/fusioninventory/files/repository/";
```

Comment:
Should not paths be declared elsewhere? I guess same ones are used on the upload part; it would be better to have only one var to change/adapt for both of them.